mathics_django/web/media/js/mathics.js

```diff
@@ -310,7 +310,14 @@ function convertMathGlyphs(dom) {
 }
 
 function createLine(value) {
-  if (value.startsWith('<math')) {
+	if (value.startsWith('<script')) {
+		var dom = document.createElement('div');
```

Comment:
@TiagoCavalcanteTrindade when I look at: 

```
Graphics3D[Polygon[{{0, -1, 0}, {2, 3, 2}, {1, 0, 2}, {3, -2, 0}}], Axes->True]
```

The axes can flow off to the left past the boundary. Maybe the image can be shifted to the right a bit? Or some parameter of the size is mismatching the size of the enclosing object and is larger? 

What do you think?

Reply:
@rocky it's happening because of the aligning is made by MathML's CSS.

Reply:
@rocky the div is centered but there's an error of the position of the tick number, working on it now.

Reply:
Thanks for being on top of this.

Reply:
@rocky I fixed the tick numbers, but now the magic numbers 10 and -8 need to be changed (graphics3d.js, 571-572). Maybe you could give some help here?

Reply:
I looked at this, and I really don't have a clue. It looks like Angus Griffith set this back in 2012 before people knew about how to set variables in javascript or how to write comments communicating what's up here. 

Also, I gotta get to work now. Catch you in about 8 hours or so.

Reply:
@rocky I tested some values and -8, -8 looks very close to the original.